src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2507,6 +2507,7 @@ namespace NServiceBus.Settings
         public SettingsHolder() { }
         public void ApplyTo<T>(NServiceBus.ObjectBuilder.IComponentConfig config) { }
         public void ApplyTo(System.Type componentType, NServiceBus.ObjectBuilder.IComponentConfig config) { }
+        public void Clear() { }
```

Comment:
We only expose the IReadOnlySettings at runtime (unless the user is smart enough cast it to `SettingHolder`)